### ISSUE TITLE
Doesn't support multiple values for keys - continued

### ISSUE THREAD

**Issue by JoBoDo** [JankGaming/vdfjs#4]
It appears that VDFJS doesn't put key/values into an array in all circumstances

I looked through your code and it appears that you only check for an array if  your "levelRegex" is matched
Below is an excerpt of VDF that has this issue.

Only "particles/arcanist/essenceshift.vpcf" is retained when the data is translated to json.
I've never raised issues before.. hope this makes sense! Let me know if there are any problems.

Example of VDF:

```
"npc_dota_hero_arcanist"
    {
        "precache"
        {
            "particle_folder"       "particles/arcanist"
            "particle"              "particles/arcanist/arcanebarrage.vpcf"
            "particle"              "particles/units/heroes/hero_lion/lion_spell_mana_drain.vpcf"
            "particle"              "particles/arcanist/arcanepulse_barrier.vpcf"
            "particle"              "particles/arcanist/arcanepulse.vpcf"
            "particle"              "particles/arcanist/essenceshift.vpcf"
        }
    }
```

Example of JSON:

```
{
  "npc_dota_hero_arcanist": {
    "precache": {
      "particle_folder": "particles/arcanist",
      "particle": "particles/arcanist/essenceshift.vpcf"
    }
  }
}
```

**Comment by twisterghost** [comment]
👍 thanks for logging this. I'll take a look soon.

**Comment by twisterghost** [comment]
Fixed in [2.0.1](https://github.com/JankGaming/vdfjs/releases/tag/v2.0.1)

**Comment by JoBoDo** [comment]
Thank you so much for fixing these so quickly! It really means the world to me :)
